calcite-qml/demo/qml/main.qml

```diff
@@ -217,10 +217,15 @@ ApplicationWindow {
                 BusyIndicator {
                     Layout.alignment: Qt.AlignHCenter
                 }
+
                 Label {
                     text: "Loading..."
                     Layout.alignment: Qt.AlignHCenter
                 }
+                SpinBox {
+                    Layout.alignment: Qt.AlignHCenter
```

Comment:
The `height` property is ignored. To change the size, user will need to use the `implicitHeight`. Is that a wanted behavior? It can be fixed otherwise

Reply:
`ColumnLayout` sizes and positions its content so I am not surprised `height` does not work in this context. It should work for a `SpinBox` which is anchored to its parent without a layout-based parent.

Reply:
`ColumnLayout` positions and sizes its contents, so `height` is controlled by the parent.

If you create a `SpinBox` outside of a layout I suspect you'll find `height` works then.

Reply:
It does